### DISCUSSION TITLE
Fontawesome Pro

### DISCUSSION THREAD

**Discussion by AvianAeternum** [requarks/wiki#6467] [Help / Questions]
I pay for Fontawesome Pro and would like to use it in wiki.js. By default, using the stylesheet script in the theme using fontawesomes cdn does not work, but seems like the easiest way to implement support for fa pro. Has anyone else managed to get fa pro to work on wiki.js?

Ive searched for similar topics, there is one other topic named identically, but is made by an engineer at FA talking about seeing some pro icons in use in default areas of the wiki, and not helpful in regards to actually using FA pro as a hoster of wiki.js

**Comment by NGPixel** [comment]
https://requarks.canny.io/wiki/p/font-awesome-pro

**Reply by AvianAeternum** [reply]
That was 3 years ago.
